<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt Pods</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/system-upgrade.svg</normaloff>:/icons/icons/system-upgrade.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QComboBox" name="comboBoxCurrentRepository">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonRemoveRepository">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Untrack selected repository. The repository will not be removed on disk.</string>
        </property>
        <property name="text">
         <string>Untrack repository</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/icons/stock_close.svg</normaloff>:/icons/icons/stock_close.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAddRepository">
        <property name="text">
         <string>Add repository</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/icons/list-add.svg</normaloff>:/icons/icons/list-add.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidgetPods">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <widget class="QWidget" name="tabLocalPods">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/icons/computer.svg</normaloff>:/icons/icons/computer.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Local repositories</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="tableViewLocal">
          <property name="toolTip">
           <string>List of all locally installed pods in the current repository.</string>
          </property>
          <property name="statusTip">
           <string>List of all locally installed pods in the current repository.</string>
          </property>
          <property name="whatsThis">
           <string>List of all locally installed pods in the current repository.</string>
          </property>
          <property name="alternatingRowColors">
           <bool>false</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="textElideMode">
           <enum>Qt::ElideLeft</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>150</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>Pods with an italic font did not pass the validation check.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditSearchLocal">
          <property name="toolTip">
           <string>Type to filter the list. You can use wildcards, ie. &quot;pack*&quot;.</string>
          </property>
          <property name="statusTip">
           <string>Type to filter the list. You can use wildcards, ie. &quot;pack*&quot;.</string>
          </property>
          <property name="whatsThis">
           <string>Type to filter the list. You can use wildcards, ie. &quot;pack*&quot;.</string>
          </property>
          <property name="placeholderText">
           <string>Filter</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButtonRemoveLocalPods">
            <property name="toolTip">
             <string>Remove selected pods from the current repository.</string>
            </property>
            <property name="statusTip">
             <string>Remove selected pods from the current repository.</string>
            </property>
            <property name="whatsThis">
             <string>Remove selected pods from the current repository.</string>
            </property>
            <property name="text">
             <string>Remove pods</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/icons/stock_close.svg</normaloff>:/icons/icons/stock_close.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonUpdateLocalPods">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="toolTip">
             <string>Attempt to update all pods in the current repository.</string>
            </property>
            <property name="statusTip">
             <string>Attempt to update all pods in the current repository.</string>
            </property>
            <property name="whatsThis">
             <string>Attempt to update all pods in the current repository.</string>
            </property>
            <property name="text">
             <string>Update pods</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/icons/system-upgrade.svg</normaloff>:/icons/icons/system-upgrade.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonRefreshLocalPods">
            <property name="toolTip">
             <string>Refresh list of local repositories manually.</string>
            </property>
            <property name="statusTip">
             <string>Refresh list of local repositories manually.</string>
            </property>
            <property name="whatsThis">
             <string>Refresh list of local repositories manually.</string>
            </property>
            <property name="text">
             <string>Refresh</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/icons/rotate.svg</normaloff>:/icons/icons/rotate.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabAvailablePods">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/icons/network_local.svg</normaloff>:/icons/icons/network_local.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Available pods</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButtonSources">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Manage pod sources.</string>
            </property>
            <property name="statusTip">
             <string>Manage pod sources.</string>
            </property>
            <property name="whatsThis">
             <string>Manage pod sources.</string>
            </property>
            <property name="text">
             <string>Manage Sources</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/icons/system-run.svg</normaloff>:/icons/icons/system-run.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonInstallExternalPod">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Install a pod that is not provided in the sources list.</string>
            </property>
            <property name="statusTip">
             <string>Install a pod that is not provided in the sources list.</string>
            </property>
            <property name="whatsThis">
             <string>Install a pod that is not provided in the sources list.</string>
            </property>
            <property name="text">
             <string>Install external pod</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/icons/file_new.svg</normaloff>:/icons/icons/file_new.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tableViewRemote">
          <property name="toolTip">
           <string>List of all available pods provided by all source.</string>
          </property>
          <property name="statusTip">
           <string>List of all available pods provided by all source.</string>
          </property>
          <property name="whatsThis">
           <string>List of all available pods provided by all source.</string>
          </property>
          <property name="alternatingRowColors">
           <bool>false</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="textElideMode">
           <enum>Qt::ElideLeft</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>150</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditSearchRemote">
          <property name="toolTip">
           <string>Type to filter the list. You can use wildcards, ie. &quot;pack*&quot;.</string>
          </property>
          <property name="statusTip">
           <string>Type to filter the list. You can use wildcards, ie. &quot;pack*&quot;.</string>
          </property>
          <property name="whatsThis">
           <string>Type to filter the list. You can use wildcards, ie. &quot;pack*&quot;.</string>
          </property>
          <property name="placeholderText">
           <string>Filter</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonRefreshAvailablePods">
            <property name="toolTip">
             <string>Refresh list of available pods manually.</string>
            </property>
            <property name="statusTip">
             <string>Refresh list of available pods manually.</string>
            </property>
            <property name="whatsThis">
             <string>Refresh list of available pods manually.</string>
            </property>
            <property name="text">
             <string>Refresh</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/icons/rotate.svg</normaloff>:/icons/icons/rotate.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonInstallPods">
            <property name="toolTip">
             <string>Install all selected pods to the current repository.</string>
            </property>
            <property name="statusTip">
             <string>Install all selected pods to the current repository.</string>
            </property>
            <property name="whatsThis">
             <string>Install all selected pods to the current repository.</string>
            </property>
            <property name="text">
             <string>Install</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/icons/list-add.svg</normaloff>:/icons/icons/list-add.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabDiagnostic">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/icons/gnome-system.svg</normaloff>:/icons/icons/gnome-system.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Diagnostic</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Build:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditBuild">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Ubuntu Mono</family>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonReportIssue">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Report an issue to the qt-pods bugtracker.</string>
            </property>
            <property name="statusTip">
             <string>Report an issue to the qt-pods bugtracker.</string>
            </property>
            <property name="whatsThis">
             <string>Report an issue to the qt-pods bugtracker.</string>
            </property>
            <property name="text">
             <string>Report an issue</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/icons/remove.svg</normaloff>:/icons/icons/remove.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEditDiagnostic">
          <property name="font">
           <font>
            <family>Ubuntu Mono</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPlainTextEdit {
	background: black;
	color: white;
}</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
